niap-cc/Permissions/Tester/app/src/normal/AndroidManifest.xml

```diff
@@ -1467,5 +1532,26 @@
     <uses-permission android:name="com.vzw.DEFAULT_APP_CHANGE" />
     <uses-permission android:name="com.vzw.engage.service.myverizon" />
     <uses-permission android:name="com.vzw.hss.myverizon.CountlyPush.BROADCAST_PERMISSION" />
+    <!-- non platform shiba(piexel8) 35 -->
```

Comment:
Here and the platform manifest below: Devices should be referenced by their model. Also nit: typo pixel.

Reply:
Fixed

Reply:
My view still shows this line as it was before; was the commit with the update included in this pull request?

Reply:
How about this time?